packages/mock-server/src/worker/handlers/masterList/index.ts

```diff
@@ -0,0 +1,14 @@
+import { mockMasterListsQuery } from '@openmsupply-client/common/src/types';
+import { ResolverService } from '../../../api/resolvers';
+
+const masterListsQuery = mockMasterListsQuery((req, res, ctx) => {
+  return res(
+    ctx.data({
+      // trouble typing the response correctly.. gave up. it's only mocks after all
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      masterLists: ResolverService.masterList.list(req?.variables) as any,
```

Comment:
this gave me grief.. decided eventually that I didn't care enough to spend more time on it

Reply:
Yeah, super annoying!